workspaces/jupyterlab-python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jupyter/minimal-notebook:2023-10-20
+FROM quay.io/jupyter/minimal-notebook:2025-02-24
```

Comment:
https://hub.docker.com/r/jupyter/minimal-notebook calls out that the Docker Hub imaage is no longer updated. https://github.com/jupyter/docker-stacks shows that the `quay.io` versions are what we should use instead.

Reply:
Maybe add a comment?

Reply:
Just noticed you already added to merge queue. This is non-blocking, feel free to ignore or push to a future PR.